flocker/apiclient/_client.py

```diff
@@ -538,6 +606,44 @@ def version(self):
             b"GET", b"/version", None, {OK}
         )
 
+    def _parse_configuration_container(self, container_dict):
+        """
+        Convert a dictionary decoded from JSON with a container's
+        configuration.
+
+        :param container_dict: Dictionary describing a container.
+        :return: ``Container`` instance.
+        """
+        return Container(
+            node_uuid=UUID(hex=container_dict[u"node_uuid"], version=4),
```

Comment:
:grey_question: Probably okay as is, just noting that this can generate `KeyError`'s if the container_dict doesn't meet the expected schema. Not for this PR, but perhaps we should re-use some of our fancy API schema yaml files to do proper type-checking on things returned from the server to make nicer errors. Or maybe I'm just missing where we do that.